okhttp-clients/src/main/java/com/palantir/conjure/java/okhttp/OkhttpTraceInterceptor.java

```diff
@@ -37,6 +37,10 @@
 
     @Override
     public Response intercept(Chain chain) throws IOException {
+        Tags.AttemptSpan attemptSpanTag = chain.request().tag(Tags.AttemptSpan.class);
+        if (attemptSpanTag == null) {
```

Comment:
Is seems like the invariant imagined by the author of this code is `every request has an AttemptSpan tag`. I think we should either try to restore this invariant or or minimally explain here why some requests have such a tag and others don't.

Reply:
Just copied this fix from [here](https://github.com/palantir/conjure-java-runtime/pull/908/files#diff-c6734fe87380e54de3ec02c05aa1f1e7R65). Happy to add a comment or more logging. Re: restoring the invariant, not really sure how to do that - maybe @dansanduleac who wrote https://github.com/palantir/conjure-java-runtime/pull/1205 has some ideas.

Reply:
it's safe to reuse it, the value can never change - you can't alter tags once the Request has been created